.github/workflows/pr_master_test.yml

```diff
@@ -638,6 +638,7 @@ jobs:
         fi
 
     - name: Upload codecov reports
+      if: github.repository_owner == 'Pyomo' || github.ref != 'refs/heads/main'
```

Comment:
For consistency, can we add that "if" to both workflow drivers?  It would be a noop for `push_branch_test.yml`, but would keep the two files in sync.